src/libraries/Common/src/System/Net/Http/Http2/IHttpHeadersHandler.cs

```diff
@@ -17,6 +17,8 @@ namespace System.Net.Http
 #endif
     interface IHttpHeadersHandler
     {
+        void OnStaticIndexedHeader(int index);
```

Comment:
Did @Tratcher chat with you about the issues with these added interface definitions?